Themes/default/languages/ManageSettings.english.php

```diff
@@ -131,6 +131,9 @@
 $txt['memcached_cache'] = 'Memcached';
 $txt['zend_cache'] = 'Zend Platform/Performance Suite';
 $txt['xcache_cache'] = 'XCache';
+$txt['cache_xcache'] = 'XCache settings';
```

Comment:
cache_xcache vs xcache_cache it's from my pov to near,
i would like to see somthing like that cache_xcache -> xcache_setting
to keep inline with the other fields and let to be clear.